### PR TITLE
Add Economize

### PR DESCRIPTION
**Pull request by HarshCasper**
Website: [Economize](economize.cloud)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Name | Twitter | What we do with Vue.js | Contact Person | Open for Event Spaces
 ------------ | ------- | ------- | ------- | ------- | -------
 [BeautifulCode](https://www.beautifulcode.co) | [@BeautifulCode_](https://twitter.com/beautifulcode_) | Remote based Service Company using Vue with Typescript | ➖ | ➖ | ➖ 
 [Clarity Technologies](https://claritytech.io/) | ➖ | Software and Services Startup | [Manash Sonowal](https://twitter.com/msonowal) | ➖ | ✅
+[Economize](https://www.economize.cloud/) | [@economizecloud](https://twitter.com/economizecloud) | Analyze, manage, and optimize your cloud infrastructure costs & billing better with Economize. | [anirudh@economize.cloud](https://github.com/anirudhmurali) | ➖ | ✅
 [DeepSource](https://deepsource.io) | [@DeepSourceHQ](https://twitter.com/DeepSourceHQ/) | Automated code review that helps you write better code. Backed by Y Combinator. | [shivam@deepsource.io](https://github.com/scmmishra) | ➖ | ✅
```
Comment (vinayakkulkarni) [inline README.md]:
```suggestion
[DeepSource](https://deepsource.io) | [@DeepSourceHQ](https://twitter.com/DeepSourceHQ/) | Automated code review that helps you write better code. Backed by Y Combinator. | [shivam@deepsource.io](https://github.com/scmmishra) | ➖ | ✅
[Economize](https://www.economize.cloud/) | [@economizecloud](https://twitter.com/economizecloud) | Analyze, manage, and optimize your cloud infrastructure costs & billing better with Economize. | [anirudh@economize.cloud](https://github.com/anirudhmurali) | ➖ | ✅
```

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
Hi @vinayakkulkarni 

I have fixed the same. Can you please have a look again :smile:

**Review by vinayakkulkarni — APPROVED** [review]
LGTM

**Comment by anirudhmurali** [comment]
Thanks Vinay! If you happen to know good Vue.js developers, please let us know! :)

**Pull request merged**